cinema/views.py

```diff
@@ -2,9 +2,17 @@
 
 from django.db.models import F, Count
 from rest_framework import viewsets
+from rest_framework.authentication import TokenAuthentication
+from rest_framework.exceptions import MethodNotAllowed
 from rest_framework.pagination import PageNumberPagination
 
 from cinema.models import Genre, Actor, CinemaHall, Movie, MovieSession, Order
+from cinema.mixins import PermissionMixin
+
```

Comment:
Do not make a blank line in case imports belong to similar import group according to PEP-8